src/dlgProfilePreferences.cpp

```diff
@@ -187,60 +187,34 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pParentWidget, Host* pHost
 
     // Set the tooltip on the containing widget so both the label and the
     // control have the same tool-tip:
-    widget_timerDebugOutputMinimumInterval->setToolTip(tr("<p>A timer with a short interval will quickly fill up the <i>Central Debug Console</i> "
-                                                          "windows with messages that it ran correctly on <i>each</i> occasion it is called.  This (per profile) "
-                                                          "control adjusts a threshold that will hide those messages in just that window for those timers which "
-                                                          "run <b>correctly</b> when the timer's interval is less than this setting.</p>"
-                                                          "<p><u>Any timer script that has errors will still have its error messages reported whatever the setting.</u></p>"));
-
-    pushButton_showGlyphUsage->setToolTip(utils::richText(tr("This will bring up a display showing all the symbols used in the current "
-                                                             "map and whether they can be drawn using just the specified font, any other "
-                                                             "font, or not at all.  It also shows the sequence of Unicode <i>code-points</i> "
-                                                             "that make up that symbol, so that they can be identified even if they "
-                                                             "cannot be displayed; also, up to the first thirty two rooms that are using "
-                                                             "that symbol are listed, which may help to identify any unexpected or odd cases.")));
+    //: Tooltip for timer debug output minimum interval
+    widget_timerDebugOutputMinimumInterval->setToolTip(tr("<p>Hide success messages in Central Debug Console for timers with intervals below this threshold. "
+                                                          "Error messages always display. "
+                                                          "<a href='https://wiki.mudlet.org/w/Manual:Timers'>Learn more</a></p>"));
```

Comment:
Test to see if
- clicking a link inside a tooltip does actually work on all OS
- screen readers will announce this in a way that is useful